AUSScoreCore/Sources/ScoresFeature/ScoresContainer.swift

```diff
@@ -45,13 +45,19 @@ public struct ScoresFeature: ReducerProtocol {
       case .task:
         return .task {
           await .dateWithGamesResponse(TaskResult {
-            let dates = try await dbClient.datesWithGames()
+            var dates = try await dbClient.datesWithGames()
+            if (!dates.contains(where: { $0.startOfDay == .now.startOfDay})) {
```

Comment:
Could you make use of `isDateInToday`?
https://developer.apple.com/documentation/foundation/calendar/2293243-isdateintoday